Examples/outputSinDAC.py

```diff
@@ -1,3 +1,8 @@
+from __future__ import division
+from __future__ import print_function
```

Comment:
LabJackPython supports Python 2.5 (see [here](https://github.com/labjack/LabJackPython/blob/3f585a88315d1d9c937ef6b7d0e80c7cd7299e91/setup.py#L4-L5) and [here](https://github.com/labjack/LabJackPython/blob/3f585a88315d1d9c937ef6b7d0e80c7cd7299e91/setup.py#L21)).  [`print_function`](https://docs.python.org/2/library/__future__.html) requires Python 2.6.  

I'm in favor of increasing the minimum required version to Python 2.6 because it would allow us to remove hacks like [`sys.exc_info`](https://github.com/labjack/LabJackPython/blob/3f585a88315d1d9c937ef6b7d0e80c7cd7299e91/src/u3.py#L1719).  Python 2.6 would also mean [`bytearray`](https://docs.python.org/2/library/functions.html#bytearray) is available so [inconsistent return values](https://github.com/labjack/LabJackPython/blob/3f585a88315d1d9c937ef6b7d0e80c7cd7299e91/src/u3.py#L1081-L1082) could be fixed.  However, I think if Python 2.5 support is to be removed, it should be removed in a separate commit that also removes the hacks used to support it.